browser/metrics/pref_names.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_METRICS_PREF_NAMES_H_
+#define BRAVE_BROWSER_METRICS_PREF_NAMES_H_
+
+class PrefRegistrySimple;
+
+namespace metrics::prefs {
+inline constexpr char kDontAskForCrashReporting[] =
```

Comment:
nit: for consistency

```suggestion
namespace metrics::prefs {

inline constexpr char kDontAskForCrashReporting[] =
```